package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Launch rule engine which processes rules on client websites and delegates logic to extensions.",
   "license": "Apache-2.0",
   "scripts": {
-    "build": "rollup -c && uglifyjs dist/engine.js -c -m -o dist/engine.min.js",
+    "build": "rollup -c --intro \"if (!window.atob) { console.warn('Adobe Launch is unsupported in IE 9 and below.'); return; }\" && uglifyjs dist/engine.js -c -m -o dist/engine.min.js",
```

Comment:
I can't just put the browser check at the top of index.js because Rollup hoists all imported module code to the top of the self-invoking function that it creates. Module code would be executed before the browser check, which we want to avoid. This intro flag puts the browser check as the first thing inside the self-invoking function.